winrm/http.go

```diff
@@ -45,16 +43,13 @@ func Http_post(client *Client, request *soap.SoapMessage) (response string, err
 	req.SetBasicAuth(client.username, client.password)
 	resp, err := httpClient.Do(req)
 	if err != nil {
-		err = fmt.Errorf("unknown error %s", err)
-		return
+		return "", fmt.Errorf("unknown error %s", err)
 	}
 
-	if resp.StatusCode == 200 {
-		response, err = body(resp)
-	} else {
-		body, _ := ioutil.ReadAll(resp.Body)
-		err = fmt.Errorf("http error: %d - %s", resp.StatusCode, body)
+	if resp.StatusCode != 200 {
+		return "", fmt.Errorf("http error: %d", resp.StatusCode)
```

Comment:
it will be fine if we included some better error handling, good tests and also for troubleshooting we can use anyway a debugger like gdb/delve or godebug.